Frameworks/Foundation/NSIndexPath.mm

```diff
@@ -38,22 +49,36 @@ + (NSIndexPath*)indexPathWithIndexes:(const NSUInteger*)indexes length:(NSUInteg
  @Status Interoperable
 */
 + (NSIndexPath*)indexPathForRow:(NSInteger)row inSection:(NSInteger)section {
```

Comment:
`indexPathForRow:inSection:` is supposed to be a category method implemented in UIKit.